test/helpers/voting.ts

```diff
@@ -40,3 +41,82 @@ export async function cancelProposal(context: DevModeContext, proposal: number)
   ]);
   expectSubstrateEvent(block, "referenda", "Cancelled");
 }
+
+export const OPEN_TECHNICAL_COMMITTEE_MEMBERS: KeyringPair[] = [alith, baltathar];
+export const OPEN_TECHNICAL_COMMITTEE_THRESHOLD = Math.ceil(
+  (OPEN_TECHNICAL_COMMITTEE_MEMBERS.length * 5) / 9
+);
+
+export const executeExtViaOpenTechCommittee = async <
+  Call extends SubmittableExtrinsic<ApiType>,
+  ApiType extends ApiTypes
+>(
+  context: DevModeContext,
+  extrinsic: Call | string,
+  voters: KeyringPair[] = OPEN_TECHNICAL_COMMITTEE_MEMBERS,
+  threshold: number = OPEN_TECHNICAL_COMMITTEE_THRESHOLD
+) => {
+  const openTechCommitteeProposal = context.pjsApi.tx.openTechCommitteeCollective.propose(
+    threshold,
+    extrinsic,
+    100
+  );
+  const { result: result2 } = await context.createBlock(openTechCommitteeProposal, {
+    signer: voters[0],
+  });
+  if (!result2?.events) {
+    throw new Error("No events in block");
+  }
+
+  let openTechProposal: `0x${string}` | undefined;
+  let openTechProposalIndex: number | undefined;
+
+  filterAndApply(result2.events, "openTechCommitteeCollective", ["Proposed"], (found) => {
+    openTechProposalIndex = (found.event as any).data.proposalIndex.toNumber();
+    openTechProposal = (found.event as any).data.proposalHash.toHex();
+  });
+
+  if (typeof openTechProposal === "undefined" || typeof openTechProposalIndex === "undefined") {
+    console.error("Error submitting OpenTechCommittee proposal");
+    return result2;
+  }
+
+  console.log(
+    `🏛️ OpenTechCommittee proposal submitted with proposal id: 
+    ${openTechProposalIndex} and hash: ${openTechProposal.slice(0, 6)}...${openTechProposal.slice(
+      -4
+    )}`
+  );
+
+  // Vote on it
+  for (const voter of voters) {
+    const nonce = (await context.pjsApi.query.system.account(voter.address)).nonce.toNumber();
+    const vote = context.pjsApi.tx.openTechCommitteeCollective
+      .vote(openTechProposal, openTechProposalIndex, true)
+      .signAsync(voter, { nonce });
+
+    await context.createBlock(vote);
```

Comment:
maybe construct an array of all signed txs and then do them in a single (not sure it matters though)?

Reply:
it doesn't matter because the number of votes is low (< 10)